openslides/motions/static/js/motions/motion-services.js

```diff
@@ -0,0 +1,121 @@
+(function () {
+
+"use strict";
+
+angular.module('OpenSlidesApp.motions.motionservices', ['OpenSlidesApp.motions', 'OpenSlidesApp.motions.lineNumbering'])
+
+    .service('MotionInlineEditing', [
```

Comment:
Is there a reason why you use a service and not a factory? The difference is, I think, that a service uses the returned object as constructor and uses `new` whereas a factory just returns the object. We do not need the `new` construction here, do we?